carla/recourse_methods/catalog/clue/library/clue_ml/src/compas_loader.py

```diff
@@ -0,0 +1,249 @@
+from __future__ import division
+
+import datetime
+import os
+
+# Python 2
+# try:
+#    import urllib2
+# except:
+#    import urllib3
+# Python 3
+import urllib.request
+from collections import defaultdict
+from random import seed
+
+import numpy as np
+import pandas as pd
+from sklearn import preprocessing
+from sklearn.model_selection import train_test_split
+
+from .UCI_loader import unnormalise_cat_vars
+
+# TODO return mean and std for variables + train test split
+
+
+def check_data_file(fname):
+    files = os.listdir(".")  # get the current directory listing
+    print
+    "Looking for file '%s' in the current directory..." % fname
+
+    if fname not in files:
+        print
+        "'%s' not found! Downloading from GitHub..." % fname
+        addr = "https://raw.githubusercontent.com/propublica/compas-analysis/master/compas-scores-two-years.csv"
```

Comment:
Do we need this whole compas loader class? I'd rather only support the current way of passing dataframes

Reply:
Yeah you're definitely right 😅 I'll have a look into that repository and try to filter everything not used out